src/wp-includes/rest-api/endpoints/class-wp-rest-menus-controller.php

```diff
@@ -52,6 +57,18 @@ public function get_item_permissions_check( $request ) {
 		return $this->check_has_read_only_access( $request );
 	}
 
+	public function check_read_permission( $term_id ) {
+		$term = $this->get_term( $term_id );
+
+		if ( ! is_wp_error( $term ) ) {
+			if ( isset( $term->show_in_rest ) && $term->show_in_rest ) {
+				return true;
+			}
+		}
+
+		return false;
```

Comment:
Nitpick: I wonder if this should be refactored to:
```suggestion
		return ! is_wp_error( $term ) && ! empty( $term->show_in_rest );
```

Reply:
Same comment as above, but I would probably just combine the conditionals to remove some nesting here.